app/actions/remote/team.ts

```diff
@@ -167,8 +167,7 @@ export const fetchAllTeams = async (serverUrl: string, fetchOnly = false): Promi
     }
 
     try {
-        const teams = await client.getTeams();
-
+        const teams = await client.getMyTeams();
```

Comment:
There were some warnings of empty team list on startup.

`getTeams` get all public teams. If no public teams, even if I am a member of some, this will return an empty list.
`getMyTeams` get all the teams I am a member of. For this case I think this is what we actually want.

Reply:
no we actually want the other one, as this is to join channels you are not a member of, right?

what we need to do is to skip the handlers if the response is an empty string

Reply:
You are right :sweat_smile: